languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -3676,6 +3676,18 @@ though, and has since been slightly modified)-->
     </rulegroup>
     <rulegroup id="SUR_ACCENT" name="Sur au lieu de Sûr">
       <url>http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2232</url>
+      <rule default="temp_off">
+          <pattern>
+             <token postag="SENT_START"/>
+             <token>Bien</token>
```

Comment:
FYI: tokens are always case-insensitive. This token could also be `<token>bien</token>` and would match the exact same words (bien, Bien, BIEN, etc.). When it's important, you can use `case_sensitive="yes"`, but in this case, I wouldn't recommend it (users don't always start their sentence with an uppercase letter).